pyproject.toml

```diff
@@ -87,6 +87,7 @@ filterwarnings = [
     "ignore::DeprecationWarning:importlib*",
     "ignore::DeprecationWarning:jsii*",
 ]
+ignore = ["tests/test-data"]
```

Comment:
It is still flagging the other files. Maybe easiest is to move these to an `examples/ialirt` directory, because it isn't really run in our test suite? That way pytest wouldn't pick them up

You could also add `norecursedirs` or `addopts=--ignore` flags instead.